### PR TITLE
charts: point out that the repo isn't set up yet

### PR DESCRIPTION
**Pull request by mrueg**
**What this PR does / why we need it**:
Users wonder why the chart repo is not available yet.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
See also: https://github.com/kubernetes/kube-state-metrics/issues/1342

### CODE REVIEW

File: charts/kube-state-metrics/README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 Installs the [kube-state-metrics agent](https://github.com/kubernetes/kube-state-metrics).
 
+## Disclaimer
```
Comment (lilic) [inline charts/kube-state-metrics/README.md:5]: I would suggest Warning instead?

**Review by lilic — COMMENTED** [review]
Thanks for the PR!

File: charts/kube-state-metrics/README.md (hunk 1 above)
Comment (mrueg) [reply charts/kube-state-metrics/README.md]: Sure, changed it!

File: charts/kube-state-metrics/README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 Installs the [kube-state-metrics agent](https://github.com/kubernetes/kube-state-metrics).
 
+## Warning
+
+**The integration is still work in progress. As of now, the mentioned repositories below are not fully set up yet.
+You can still use the deprecated [helm/stable](https://charts.helm.sh/stable/) repository and migrate at a later point to the new location.
+At this point in time, no significant changes to the chart were made in this repository.
+For more information, please see: https://github.com/kubernetes/kube-state-metrics/pull/1325#issuecomment-749732052**
```
Comment (scottrigby) [inline charts/kube-state-metrics/README.md:10]:
```suggestion
## ⚠️ Warning

**Chart Releasing workflow is currently broken due to an upstream issue with this repo and `kubernetes/test-infra`, which is actively being worked on.**
Until then, the mentioned repositories below are not fully set up yet.
You can still use the deprecated [helm/stable](https://charts.helm.sh/stable/) repository and migrate to the new location in the near future.
At this point in time, no significant changes to the chart were made in this repository.
For more information or to follow progress, see: https://github.com/kubernetes/kube-state-metrics/pull/1325#issuecomment-749732052
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
Suggestions: Warning icon for quick visual spotting. Content changes for clarity. Bolding the first sentence only, so it still stands out but overall easier reading the details.

**Review by scottrigby — APPROVED** [review]

**Review by lilic — COMMENTED** [review]
/lgtm

Thanks!

**Pull request merged**